admin/sales_analytics/keen.py

```diff
@@ -0,0 +1,25 @@
+from admin.base.settings import KEEN_PROJECT_ID, KEEN_READ_KEY, KEEN_WRITE_KEY
+
+KEEN_CREDENTIALS = {
+    'keen_ready': False
+}
+
+if KEEN_CREDENTIALS['keen_ready']:
+    KEEN_CREDENTIALS.update({
+        'keen_project_id': KEEN_PROJECT_ID,
+        'keen_read_key': KEEN_READ_KEY,
+        'keen_write_key': KEEN_WRITE_KEY
+    })
+
+
+# TODO: when Keen is ready, use of settings instead
```

Comment:
Shouldn't settings be in `defaults.py` There is certainly a way that the credentials chain can work with defaults and local.py in the admin and the OSF.